client/stylesheets/base.less

```diff
@@ -2605,9 +2605,8 @@ a.github-fork {
 			text-overflow: ellipsis;
 			position: relative;
 			padding-left: 10px;
-			color: @secondary-font-color;
-			.calc(width,
-			~"100% - 45px");
+			//color: @secondary-font-color;
```

Comment:
Do we want the whole name colored according to status, or just rely on the dot on the left of every name entry?